src/block/context.rs

```diff
@@ -0,0 +1,650 @@
+//! The Context module holds all the functionality for the test declaration, that is:
+//! `before`, `after`, `suite`, `context`, `it` and their variants.
+//!
+//! A Context can also holds reference to children Contextes, for whom the before closures will be
```

Comment:
- from whom
- `_before_` (it's a special word)
- s/closures/functions would be clearer for the newcomer